src/components/Patient/PatientHome.tsx

```diff
@@ -243,11 +243,7 @@ export const PatientHome = (props: any) => {
   };
 
   const isPatientEligibleForNewConsultation = (patientData: PatientModel) => {
```

Comment:
this function can be converted to a variable, also why remove the additional checks?

Reply:
The original function with the additional checks, returned false regardless of whether the patient had been discharged or not.

Reply:
[Update] The function is no longer needed as this wasn't being used anywhere apart from the `Add Consultation` button which is to be removed.